app/views/families/_family.html.erb

```diff
@@ -8,6 +8,6 @@
   <td><%= family.bus_driver %></td>
   <td><%= family.animals %></td>
   <td><%= family.active %></td>
-  <td><%= link_to t("families.index.destroy"), family_path(family), method: :delete, class: "list-button" %></td>
-  <td><%= link_to t("families.index.edit"), edit_family_path(family), class: "list-button" %></td>
+  <td><%= link_to t("families.index.destroy"), family_path(family, locale: I18n.locale), method: :delete, class: "list-button" %></td>
```

Comment:
If you look at the curriculum, there is a way to add the locale to all the links by default.
In `ApplicationController`:

```
def default_url_options(options = {})
  { locale: I18n.locale }.merge(options)
end
```

Reply:
Done with fixup in sha `631fd59`